src/SFML/Window/WindowBase.cpp

```diff
@@ -75,6 +76,37 @@ WindowBase::~WindowBase()
 }
 
 
+////////////////////////////////////////////////////////////
+WindowBase::WindowBase(WindowBase&& windowBase) noexcept :
+m_impl(std::exchange(windowBase.m_impl, nullptr)),
+m_size(std::exchange(windowBase.m_size, {}))
+{
+    // Check fullscreen style and set to new object
+    if (&windowBase == getFullscreenWindow())
+        setFullscreenWindow(this);
+
+    windowBase.close();
+}
```

Comment:
```suggestion
WindowBase::WindowBase(WindowBase&& windowBase) noexcept
{
    *this = std::move(windowBase);
}
```
I'm not totally sure if this is correct and a good idea but we can maybe implement move construction in terms of move assignment and thus deduplicate some code.

Reply:
the different is in my code resources constructed with new values but in suggested code they constructed with default and then re-assigned.